src/testbed.cu

```diff
@@ -642,6 +642,13 @@ void Testbed::imgui() {
 			GPUMemory<float> density = get_density_on_grid(res3d, aabb);
 			save_density_grid_to_png(density, fname, res3d, m_mesh.thresh);
 		}
+		ImGui::SameLine();
+		if (ColoredButton("Save RGBA PNGs", 0.2f)) {
```

Comment:
Could you use `imgui_colored_button` like above?

Reply:
Also: could you wrap this button in `if (m_testbed_mode == ETestbedMode::Nerf) { }`? The rest of the marching cubes section also works with SDFs, but the RGBA slices do not.

(I would appreciate if you could wrap the entire "Marching cubes" section in
```
if (m_testbed_mode == ETestbedMode::Nerf || m_testbed_mode == ETestbedMode::Sdf) {
}
```
while you're at it. This is an oversight from us -- it doesn't work in the other modes and shouldn't be shown in the first place.)